packages/next/next-server/server/render.tsx

```diff
@@ -504,6 +510,12 @@ export async function renderToHTML(
   // _app's getInitialProps for getServerProps if not this can be removed
   if (isDataReq) return props
```

Comment:
The below code setting `props.pageProps` to an empty object probably has to come before this.